docs/ref/migration-operations.txt

```diff
@@ -245,6 +245,12 @@ operation that adds that field and so will try to run it again).
     The ability to pass parameters to the ``sql`` and ``reverse_sql`` queries
     was added.
 
+.. versionadded:: 1.8
+
+When you don't give a ``reverse_sql``, the operation and hence the migration
```

Comment:
I'd rephrase to make it more obvious that what's `versionadded:: 1.8` is the `RunSQL.noop` part.